src/index.html

```diff
@@ -5,33 +5,37 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" type="text/css" href="styles.css">
-
-  <!-- font awesome -->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:wght@300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,700&display=swap" rel="stylesheet">
+
   
-  <title>Leaderboard</title>
+  <title>Leaderboard App</title>
 </head>
 
 <body>
-  <h1>Leaderboard</h1>
+  <h1>LeaderBoard</h1>
   <div class="container">
-    <div class="left-section">
-      <div id="recentScoresContainer">
-        <h2 id="recentScoresHeading">Recent Scores</h2>
-        <button id="refreshButton"><i class="fa fa-refresh" aria-hidden="true"></i></button>
+      <div class="recentScores">
+          <div class="recentScoreTitle">
+              <h2>Recent Scores</h2>
+              <button id="refresh" type="button"><i class="fa fa-refresh" aria-hidden="true"></i></button>
+          </div>
+          <div class="recentScoreList">
+              <ul id="list">
+               </ul>
+          </div>
+      </div>
+      <div class="addScore">
+          <form action="submit" class="add-form">
+              <h2 id="addScore">Add your score</h2>
+              <input class="input name" type="text" name="name" placeholder="Your name" required>
+              <input class="input score" type="text" name="score" placeholder="Your score" required>
```

Comment:
:orange_circle: [OPTIONAL] I  noticed that you have an HTML input field with the type="text" attribute. However, the expected input for this field is numbers. To ensure that only numbers can be entered into this field, you should change the type attribute to "number".

Here is an example of how to change the type attribute:

` <input class="input score" type="number" name="score" placeholder="Your score" required>`

This will ensure that only numbers can be entered into the score field.